oceannavigator/frontend/webpack.config.js

```diff
@@ -1,80 +1,80 @@
-const webpack = require('webpack');
-const path = require('path');
+const webpack = require("webpack");
+const path = require("path");
 
-const BUILD_DIR = path.resolve(__dirname, 'public');
-const APP_DIR = path.resolve(__dirname, 'src');
+const BUILD_DIR = path.resolve(__dirname, "public");
+const APP_DIR = path.resolve(__dirname, "src");
 
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 const config = {
-    entry: [
-        'babel-polyfill',
-        APP_DIR + '/index.jsx',
-    ],
-    output: {
-        path: BUILD_DIR,
-        filename: 'oceannavigator.js',
-        publicPath: '/public/'
-    },
-    module: {
-        rules: [
-            {
-                test: /\.jsx?$/,
-                include: APP_DIR,
-                loader: 'babel-loader'
-            },
-            {
-                test: /\.scss$/,
-                use: [
-                    MiniCssExtractPlugin.loader,
-                    "css-loader",
-                    "sass-loader"
-                ]
-            },
-            {
-                test: /\.css$/,
-                use: [MiniCssExtractPlugin.loader, "css-loader"]
-            },
-            {
-                test: /\.(png|gif|svg|eot|woff2?|ttf|svg)(\?.*)?$/,
-                loader: 'file-loader',
-                options: {
-                    name: '/[name].[ext]'
-                }
-            }
+  entry: [
+    APP_DIR + "/index.jsx",
+  ],
+  output: {
+    path: BUILD_DIR,
+    filename: "oceannavigator.js",
+    publicPath: "/public/"
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(t|j)sx?$/,
```

Comment:
Progressive migration to typescript, so all of our existing `.jsx` files will still be automatically compiled.